src/SFA.DAS.FAT.Web/Models/BreadCrumbs/IBackLink.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.FAT.Web.Models.BreadCrumbs;
+
+public interface IBackLink
+{
+    string BackLinkUrl { get; set; }
+    bool ShowBackLink { get; }
```

Comment:
This can have default implementation by calculating from existence of Back link.

Reply:
not in an interface, would need to be a concrete class